drivers/gpio/gpio_mcux.c

```diff
@@ -96,7 +96,7 @@ static int gpio_mcux_configure(const struct device *dev,
 		pcr |= PORT_PCR_PE_MASK | PORT_PCR_PS_MASK;
 
 	} else if ((flags & GPIO_PULL_DOWN) != 0) {
-		/* Enable the pull and select the pulldown resistor (deselect
+		/* Enable the pull and select the pulldown resistor deselect
```

Comment:
replace parentheses with comma instead, the grammar doesnt make sense here with no punctuation